scripts/catchup.mjs

```diff
@@ -1,201 +1,282 @@
 import 'dotenv/config';
-import fetch from 'node-fetch';
+import { execSync } from 'child_process';
 import fs from 'fs';
 import minimist from 'minimist-lite';
```

Comment:
You hsould consider using 

const commandLineArgs = require('command-line-args')
const commandLineUsage = require('command-line-usage')

They are more nice for dev experience and you can print the help text easily